components/observations/ObservationsListView.tsx

```diff
@@ -298,7 +336,7 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
     [filterConfig, setFilterConfig],
   );
 
-  if (incompleteQueryState(observationsResult, mapResult) || !mapLayer) {
+  if (incompleteQueryState(observationsResult, mapResult) || !filteredMapLayer) {
```

Comment:
The check for `!mapLayer` is to guard against that request returning nothing - let's check here for the responses, not computed values from the responses, e.g. `|| !mapLayer || !centerMetadata || !alternateZones`